cardano-testnet/src/Testnet/Util/Runtime.hs

```diff
@@ -19,29 +21,42 @@ module Testnet.Util.Runtime
   , poolNodeStdout
   , readNodeLoggingFormat
   , startNode
+  , ShelleyGenesis(..)
+  , shelleyGenesis
+  , getStartTime
+  , fromNominalDiffTimeMicro
   ) where
 
 import           Prelude
 
 import           Control.Monad
-import           Data.Aeson (FromJSON)
+import           Control.Monad.Error.Class
+import           Control.Monad.IO.Class
+import           Control.Monad.Trans.Except
+import           Data.Aeson ((.:))
+import qualified Data.Aeson as A
```

Comment:
We should decide on a convention when importing `Data.Aeson`. We could discuss this offline and make the change across the repo.